File: 2016 maj/python/1.py
with open("dane_6_1.txt", "r") as f:
    file = f.read().strip().split('\n')

def encode(s, key = 107):
    return ''.join([chr(65 + (ord(i) - 65 + key) % 26) for i in s])

with open("wyniki_6_1.txt", "w") as f:
    for s in file:
        f.write(encode(s) + '\n')

File: 2016 maj/c++/1.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;


int main()
{
    ifstream in("dane_6_1.txt");
    ofstream out("wyniki_6_1.txt");

    for (int i = 1; i <= 100; i++)
    {
        string s;
        in >> s;
        for (char c: s)
            out << char('A' + (c - 'A' + 107) % 26);
        out << '\n';
    }
}